### ISSUE TITLE
Reportovanie neplatne zaočkovaných po 1.2.2022

### ISSUE THREAD

**Issue by uhliksk** [Institut-Zdravotnych-Analyz/covid19-data#149]
Od 1. februára 2022 sa mení doba platnosti očkovania na 9 mesiacov. Akým spôsobom to ovplyvní reportované dáta? Budú zaočkovaní pred viac ako 9 mesiacmi naďalej reportovaní ako zaočkovaní alebo prepadnú do kategórie nezaočkovaní? Uvažuje sa o zavedení novej kategórie pre zaočkovaných pred viac ako 9 mesiacmi, aby bolo možné sledovať samostatne vplyv dlhého odstupu od zaočkovania? Pevne verím, že títo exspirovaní neprepadnú do nezačkovaných, lebo to by znamenalo absolútne znehodnotenie štatistických dát.

**Comment by mhudec** [comment]
Ak tymi datami myslis data o ockovani (covid19-data/Vaccination/), tak to su hlasene ockovania, cize pocet podanych davok (resp. planovane ockovania a cakaren). Ludia v hlasenych ockovaniach budu vzdy ockovani, v reportovani by sa nemali prepadnut do kategorie "nezaockovani"... do tej kategorie sa vsak mozu prepadnut z pohladu protiepidemickych opatreni, podla toho ako budu aktualne definovane tieto kategorie ludi. Clovek moze mat vyexpirovany "covid pas" (certifikat v aplikacii GreenPass atd.), ale tie podane davky mu z tela nikto nijak nevyextrahuje a nezrusi mu zaznamy v zdravotnej karte.

**Comment by MartinHBA** [comment]
@mhudec  je tam ale taky zvlastny udaj "fully" ktory sa vypocitava podla vyhlasok, nehovori o podanej davke, ale o 2 tyzdnoch po mrna a 3 tyzdne po J&J, ten asi bude reagovat aj na to co sa pyta @uhliksk , ja tento patvar vymazavam 🤣

**Comment by mhudec** [comment]
Ano, ten fully som si vsimol, preto som mal radsej prehladnejsie data v OpenData_Slovakia_Vaccination_Regions.csv, ktore boli navyse na dennej baze a rozlisovali prvu, druhu a tretiu davku (nezaujimalo ma, ci je clovek povazovany v zmysle vyhlasok za plne ockovaneho alebo nie, islo mi o pocty podanych davok), ale tam im zase chybali cisla od vseobecnych lekarov, co ockuju... nuz, ja som sa po dvoch rokoch zberu statistik na to vykaslal... :)

**Comment by uhliksk** [comment]
@mhudec Myslel som aj dáta o hospitalizáciách, kde sa uvádza v **OpenData_Slovakia_Covid_Hospital_Details.csv** a **OpenData_Slovakia_Covid_Hospital_UPV_AdmissionDischarge.csv** parameter **Vaccinated**. Bude tento parameter znamenať, že bola osoba plne zaočkovaná kedykoľvek v minulosti alebo zaočkovaní pred viac ako 9 mesiacmi tam už budú mať vo februárových dátach **FALSE**?

**Comment by mhudec** [comment]
@uhliksk - hm, v **OpenData_Slovakia_Covid_Hospital_Details.csv** nevidim nic o ockovani a v **OpenData_Slovakia_Covid_Hospital_UPV_AdmissionDischarge.csv** vidim, ze "Vaccinated" ma moznosti TRUE, FALSE a NA, tak to netusim, ze ako to myslia...

... mozem sa len dohadovat, ze ide o stav "plne zaockovany" (cize dokoncene zakladne ockovanie), "neockovany" (cize bez dokonceneho alebo vobec zacateho zakladneho ockovania) a "nezname" (bez informacii o stave ockovania). Ci sa ale dohadujem spravne, to je otazka...

**Comment by uhliksk** [comment]
@mhudec Pardon, malo to byť **OpenData_Slovakia_Covid_Hospital_AdmissionDischarge.csv** a **OpenData_Slovakia_Covid_Hospital_UPV_AdmissionDischarge.csv**. Určite by bolo zaujímavé, ak by namiesto TRUE, FALSE a NA tam bolo aj časové rozlíšenie, čiže napríklad počet mesiacov od očkovania.

**Comment by mhudec** [comment]
@uhliksk - hmmm... cim by ta informacia bola pre teba zaujimava?

**Comment by uhliksk** [comment]
> @uhliksk - hmmm... cim by ta informacia bola pre teba zaujimava?

Napríklad ak je tam len "TRUE", tak to môže byť pacient očkovaný nedávno v januári 2022 treťou dávkou, ale aj pacient očkovaný pred rokom v januári 2021 druhou dávkou. Z tohto hľadiska je už prakticky údaj o zaočkovaní hospitalizovaného zbytočný. Chýba aj info o prekonaní infekcie, atď.. Celkovo tie dáta pomaly a isto strácajú zmysel a nie je možné z nich nič relevantné vo vzťahu k očkovaniu vyhodnotiť. V čase, keď sme žili sen o dvoch dávkach na celý život to zmysel malo, ale dnes už je tento údaj nedostatočný.

**Comment by mhudec** [comment]
@uhliksk - co sa tyka vztahu k zaockovaniu, tak z toho vies vyvodit, ci ten clovek bol ockovany alebo nebol. Mimochodom, aj po uplynuti "formalnych" deviatich mesiacov sa zo zaockovaneho nestane nezaockovany, imunitny system zaockovaneho totiz bude stale poznat SARS-CoV-2 a bude ho vediet rozoznavat podla jeho antigenov (v tomto pripade spike proteinu), imunitny system nezaockovaneho vsak SARS-CoV-2 poznat nebude (a spozna ho bud pri ockovani alebo pri prekonavani ochorenia). Ale ano, "formalne" (pre potreby roznych vyhlasok) sa bude ten zaockovany po deviatich mesiacoch brat ako nezaockovany, hoci nezaockovanym v skutocnosti nebude. Tolko realita a co sa tyka interpretacie tychto konkretnych dat, tak...

@KristianSufliarsky a @matejmisik - ... prosim vas, ako je to s parametrom "Vaccinated" pri datach o hospitalizaciach? Zda sa, ze co clovek, to rozdielna interpretacia tejto informacie... tak by sa celkom zislo vediet, ze ako to vnimaju kompetentni. Dakujem.

**Comment by matejmisik** [comment]
@mhudec sorry za ultra late reply. Pri hospitalizaciach je to reported udaj pacientom v datach ktore reportuje nemocnica. Ked teraz po case nam ale pritekaju data z poistovni tak javi sa, ze pacienti niekedy povedia ze su zaockovani, pricom niesu (jedine, ze by boli v zahranici ockovani, co nebude az taky pocet). Takze pocet hospitalizacii s ockovanim bude mierne vyssi ako v skutocnosti.

Kazdopadne, pri tych self reported datach z nemocnic nepreradujeme pacienta do kategoria nezaockovany po case. Lebo presne ako pises, ten efekt ockovania nebude nulovy ani po formalnych 9 mesiacoch, aj ked bude samozrejme vyrazne znizena efektivita.
